### PR TITLE
Adapt for RPM 4.15

### PR DESCRIPTION
**Pull request by hroncok**
Fixes https://github.com/fedora-python/taskotron-python-versions/issues/86

### CODE REVIEW

**Comment by hroncok** [comment]
@kparal please let me know if you are able to review this or if I shoul try to get smebody from our team on this (@irushchyshyn unfortunately no longer works on Fedora, so you probably know more about this project than any of my python-main colleagues).

**Comment by kparal** [comment]
I have never looked deep into actual code of this task, but this patch is simple enough. I have one question, though, I'll start a review.

File: taskotron_python_versions/common.py [hunk 1]
```diff
@@ -69,6 +69,17 @@ def file_contains(path, needle):
...
 
 
+def surrogate(text):
+    """Returns unicode for a given RPM header.
+    This is a layer to support both RPM 4.14 and 4.15.
+    RPM 4.14 always returns bytes.
+    RPM 4.15 always returns unicode strings with surrogateescaped errors.
+    """
+    if isinstance(text, bytes):
+        return text.decode('utf-8', errors='surrogate')
```
Comment (kparal) [inline taskotron_python_versions/common.py]: I haven't found `surrogate` as a supported value, am I searching wrong?
https://docs.python.org/3/library/stdtypes.html#bytes.decode

Comment (hroncok) [reply taskotron_python_versions/common.py]: Good catch. it is surrogateescape, not surrogate and it is probably not properly documented:

```pycon
>>> 'ž'.encode('iso8859-2').decode('utf-8', errors='surrogateescape')
'\udcbe'
>>> 'ž'.encode('iso8859-2').decode('utf-8', errors='surrogate')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
LookupError: unknown error handler name 'surrogate'
```

Comment (hroncok) [reply taskotron_python_versions/common.py]: Actually, it is registered here https://docs.python.org/3/library/codecs.html#error-handlers

Comment (kparal) [reply taskotron_python_versions/common.py]: I found it documented under:
https://docs.python.org/3/library/codecs.html#error-handlers

**Comment by kparal** [comment]
With the latest fix, I don't see any issues, ack from me.

**Comment by hroncok** [comment]
Thanks.

File: taskotron_python_versions/common.py (hunk 1 above)
Comment (hroncok) [reply taskotron_python_versions/common.py]: Also reported this: https://bugs.python.org/issue37388

**Pull request merged**